docs/react-v9/contributing/command-cheat-sheet.md

```diff
@@ -8,6 +8,7 @@ yarn create-package # scaffolds a new package
 yarn create-component # scaffolds a new component
 yarn change # creates a new change file, if needed
 yarn clean # tidies any cached dependencies
+yarn build # generates and relinks all packages and temporary files, this can be a good option if you see errors unrelated to the packages you are working on
```

Comment:
FYI: this won't link anything. only tool that is linking workspace node_modules is `yarn`